### PR TITLE
Improve/layout UI

### PR DESCRIPTION
**Pull request by OwaisRaza**

### CODE REVIEW

File: components/header.js [hunk 1]
```diff
@@ -1,6 +1,12 @@
 import { Layout } from "antd";
+import AppNavbar from "./navbar";
 const { Header } = Layout;
 
 export default function AppHeader() {
-  return <Header className="site-layout-background" style={{ padding: 0 }} />;
+  return (
+    <Header className="site-layout-background" style={{ position: "fixed", zIndex: 99, width: "100%", padding: 0 }}>
```
Comment (DaniyalKukda) [inline components/header.js:7]: move inline css to styles folder

File: components/navbar.js [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+import AppMenuItems from "./menuItems";
+import styles from "@/styles/Navbar.module.css";
+import { useRouter } from "next/router";
+
+export default function AppNavbar() {
+  const [visible, setVisible] = useState(false);
+  const [isSelected, setIsSelected] = useState("0");
+  const router = useRouter();
+
+  const items = [
```
Comment (DaniyalKukda) [inline components/navbar.js:20]: move this items array in a separate constant file or outside the component

File: pages/sales/index.js [hunk 3]
```diff
@@ -72,7 +75,13 @@ const Sales = () => {
   if (error) return <Alert message={error} type="error" />;
   return (
     <>
-      <Title level={2}>Sales List</Title>
+      <Title level={2}>
```
Comment (DaniyalKukda) [inline pages/sales/index.js:78]: can we make this title as a separate reusable component

File: pages/purchase/index.js [hunk 4]
```diff
@@ -67,12 +78,63 @@ const PurchaseOrders = () => {
...
+        return null;
+      },
+    },
   ];
 
   if (error) return <Alert message={error} type="error" />;
   return (
     <>
-      <Title level={2}>Purchase Order List</Title>
+      <Title level={2}>
```
Comment (DaniyalKukda) [inline pages/purchase/index.js:131]: i think we need to make this title a separate reusable component

File: pages/sales/index.js (hunk 3 above)
Comment (hammadtariq) [reply pages/sales/index.js:78]: we have separate title component already with name AppTitle

**Pull request merged**